### ISSUE TITLE
question about central value

### ISSUE THREAD

**Issue by JoseBarreiros-TRI** [Denys88/rl_games#279]
@Denys88 thank you for the nice repo. 
I noticed you use a central value network when using asymmetric actor-critic. Could you please elaborate on what the central value net is exactly doing? Is this just the critic net?

**Comment by Denys88** [comment]
yes it is a critic network but with additional inputs. For example policy can have only a few parameters as obs but critic can have the whole world state because we don't use critic during inference.

**Comment by MicheleBrusadin** [comment]
@Denys88 thanks for the answer. Yes, I return both obs and state in the input dict. 
I actually edited my comment, cause I found that the issue was a missing keyword for the network builder: central_value=True

Without that keyword it was trying to build the critic network as an actor network.
